api/inc/page_allocator_exports.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __UVISOR_API_PAGE_ALLOCATOR_EXPORTS_H__
+#define __UVISOR_API_PAGE_ALLOCATOR_EXPORTS_H__
+
+#include "api/inc/halt_exports.h"
+#include <stdint.h>
+#include <stddef.h>
+
+
+#define UVISOR_ERROR_PAGE_OK                    (0)
+#define UVISOR_ERROR_PAGE_OUT_OF_MEMORY         (UVISOR_ERROR_CLASS_PAGE + 100)
```

Comment:
Since you have error classes now, it probably isn't necessary to start at 100, but if you want to that is fine.